packages/nextra-theme-docs/src/index.tsx

```diff
@@ -212,6 +213,7 @@ const InnerLayout = ({
           'menu-active': menu
         })}
       >
+        <Banner />
         {themeContext.navbar ? (
```

Comment:
I guess rendering `<Banner />` should be not depending on `themeContext.navbar` value